Duplicati/Library/Main/Controller.cs

```diff
@@ -1097,6 +1108,12 @@ public long MaxDownloadSpeed
             set { m_options.MaxDownloadPrSecond = value; }
         }
 
+        public DateTime LastCompact
+        {
+            get { return m_lastcompact; }
+            set { m_lastcompact = value; }
+        }

```

Comment:
This isn't very critical, but can we convert this to an auto-implemented property and remove the explicit `m_lastcompact`
backing field?  The behavior is the same, but it would clean up the code a very tiny bit.

Reply:
Done... let me know if that's what you were looking for.  Thanks for the suggestion!

Reply:
:+1: Thanks!

Reply:
If you think this approach is ok, I am planning on doing similar with an auto vacuum interval. Was waiting to see what you guys thought.